optionals/compat_rhs_gref3/CfgMagazines.hpp

```diff
@@ -0,0 +1,9 @@
+class CfgMagazines {
+    class CA_Magazine;
+    class rhsgref_50Rnd_792x57_SmE_drum: CA_Magazine { // drum, but still just a belt
+        ACE_isBelt = 1;
```

Comment:
On 2nd thought, 
magazine repack should be ok with these "assault drums"
but linking ammo via assistant gunner probably doesn't make sense???

Reply:
Linking ammo makes as much sense as any other belt with a starter tab.
The assistant just needs to remove the tab from the to be linked belt and then use that round to link to the end of the old belt.

The belt being inside of the drum might make it a bit of a headache, but ACE allows belt linking for other belts that feed from boxes.

Reply:
As far as I can tell the drum is attached to the weapon by a clip on the drum's door
so opening the door would either be impossible or cause the magazine to fall free

You would also be working on the bullets and links from the rear

But I'm not really sure, I'll just merge as is for now and we can adjust if someone who knows more about the weapon has some thoughts.

Reply:
The drum is removed every time the weapon is reloaded.

The assistant could just detach the drum, the remaining belt would slip out of the drum, and another belt would be linked to it and its drum attached.